homeassistant/components/netdata/sensor.py

```diff
@@ -145,6 +147,72 @@ async def async_update(self):
         )
 
 
+class NetdataAlarms(Entity):
+    """Implementation of a Netdata alarm sensor."""
+
+    def __init__(self, netdata, name, host, port):
+        """Initialize the Netdata alarm sensor."""
+        self.netdata = netdata
+        self._state = None
+        self._name = name
+        self._host = host
+        self._port = port
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self._name} Alarms"
+
+    @property
+    def state(self):
+        """Return the state of the resources."""
+        return self._state
+
+    @property
+    def device_info(self) -> Optional[Dict[str, Any]]:
+        """Return host information, e.g. the url to netdata dashboard."""
+        return {"url": f"http://{self._host}:{self._port}"}
+
+    @property
+    def entity_picture(self):
```

Comment:
I suggest setting an `icon` instead of using `entity_picture`. I think the icons are shipped with the frontend so then there's no relying on a cdn to serve the icons.

Reply:
I'd love to use local icons too. My intention with the pictures was driven by the need for colors (green, yellow, red) because with hundreds of items on an overview page, the colors are VERY helpful to see at a glance if there were any issues. Haven't found a way to apply color to icons. Is it somehow possible?

Reply:
I don't think so. Icons are mostly a frontend concern and the user can customize this if they wish. Let's change to icon.

Reply:
Changed to icons.